src/page-modules/contact/means-of-transport/means-of-transport-form-machine.ts

```diff
@@ -0,0 +1,269 @@
+import { TransportModeType } from '@atb-as/config-specs';
+import { assign, fromPromise, setup } from 'xstate';
+import { Line } from '../server/journey-planner/validators';
+import { machineEvents, Area } from '../machineEvents';
+import { InputErrorMessages, commonFieldValidator } from '../validation';
+import { convertFilesToBase64 } from '../utils';
+
+type APIParams = {
```

Comment:
I think the type name should be something with context because this type is not really what the API wants (e.g. line as a string instead of `Line`)